code-for-canada/project-thundercat
# [Inbox] Email preview

# Description

Adding preview for emails.

NOTE:
1) There is no way to mark an email as 'replied to" in the UI. However, the image does appear when its state is set correctly and I have created a test for that very thing.
2) The state will not be saved if you leave the inbox tab: it will reset all emails as not replied to or read.

## Type of change

Please delete options that are not relevant.

- New feature (non-breaking change which adds functionality)
- Code cleanliness or refactor

## Screenshot
![image](https://user-images.githubusercontent.com/2746350/55181142-7352e200-5161-11e9-997b-5527b2f23ce2.png)

![image](https://user-images.githubusercontent.com/2746350/55181163-81086780-5161-11e9-8fa1-93238cde6501.png)






# Testing

Applicable for all code changes.

Manual steps to reproduce this functionality:

1.  Go to Protype -> Start eMIB
2. Navigate through pre-test instructions
3. Start Test
4. Select the Inbox
5. Swap between the previews, make sure the "email" changes and the preview updates properly.

# Checklist

Applicable for all code changes.
